### ISSUE TITLE
[Feb 2025] Accessibility Audit: Results page user flows

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8518]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the Results page user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/results`

Use cases to test:
* view results page with no results and multiple results
* view results page with multiple pages
* result details modal
* download test results modal
* print/email/text/correct result modals

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audits #5775, #6328

**Comment by emyl3** [comment]
Findings:
`/results` multiple results - [dev.simplereport.gov-Results-multiple-results-2025-02-27.csv](https://github.com/user-attachments/files/19012768/dev.simplereport.gov-Results-multiple-results-2025-02-27.csv)

**Comment by emyl3** [comment]
Issues:
- https://github.com/CDCgov/prime-simplereport/issues/8544

**Comment by emyl3** [comment]
Ready for review!
